modules/ROOT/pages/to-publish-assets-maven.adoc

```diff
@@ -279,6 +279,32 @@ Create an `assembly.xml` file in `src/main/resources/docs-assembly`:
 </assembly>
 ----
 
+== Asset Lifecycle State
+
+A new asset version is in the `development` state or the default `stable` state.
+
+To create an asset version in the `stable` state, use Maven normally.
+
+To create an asset version in the `development` state, use the Anypoint Exchange Maven Facade API version 3 and set the version of the asset to `SNAPSHOT`.
+
+A `SNAPSHOT` asset is permanently in the `development` state and cannot be promoted to any other state.
+
+An asset with the version `SNAPSHOT` published with the Anypoint Exchange Maven Facade API version 1 or version 2 has the state `stable`.
```

Comment:
Maybe we can add that this behavior is aimed to be backwards compatible with the legacy APIs